send.go

```diff
@@ -114,7 +114,7 @@ func (c *Connection) send(ctx context.Context, request *http.Request) (response
 	switch request.Method {
 	case http.MethodGet, http.MethodDelete:
 		if request.Body != nil {
-			err = fmt.Errorf(
+			c.logger.Warn(ctx,
 				"request body is not allowed for the '%s' method",
 				request.Method,
 			)
```

Comment:
To be more consistent with other places that send log messages I'd like to have this formatted as follows:

```go
c.logger.Warn(
        ctx,
        "Request body is not allowed for the '%s' method",
        request.Method
)
```

Other than that this looks good to me.

Reply:
@jhernand Done.